### ISSUE TITLE
Deleting the first row in the logbook crashes OpenCPN

### ISSUE THREAD

**Issue by soggygeek** [rgleason/LogbookKonni_pi#69]
Deleting the first row in the logbook crashes OpenCPN.  This is easily repeatable. Deleting other rows does not crash.
Have logbook open with multiple rows. Right click on row and say Delete Row, does not crash if not first row. First row crashes. 
Windows 11 Pro, O 5.9.3 (beta) Logbook 1.4.0

**Comment by rgleason** [comment]
What version of logbook please?

**Comment by sebastien-rosset** [comment]
> What version of logbook please? Oh I see 1.4.34

No, sorry that was just an example. I was asking for the same question that you asked, and then I realized there is a problem with the version displayed in Options -> Plugins (#74)

**Comment by rgleason** [comment]
One step at a time I guess.  Thanks.
